x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_style_property.js

```diff
@@ -119,6 +138,10 @@ export class DynamicStyleProperty extends AbstractStyleProperty {
   }
 
   formatField(value) {
+    if (value === EMPTY_VALUE) {
```

Comment:
Not sure this check belongs here. What about values where empty string is valid like maybe a URL formatter? Think this needs more consideration for different types. Maybe best to just leave in original location for now